lib/chef/knife/cs_server_delete.rb

```diff
@@ -47,6 +47,11 @@ def run
 
         if server['state'] == 'Destroyed' then
           ui.warn("Server '#{hostname}' already destroyed")
+	  expunge = confirm_action("Server '#{hostname}' already destroyed, do you want to expunge it?")
+          if expunge then
+            connection.delete_server(hostname, true)
+            next
+          end
           next
```

Comment:
Hi Anton,

Just worndering, can we replace all these lines with: 
```
connection.delete_server(hostname, true) if confirm_action("Server '#{hostname}' already destroyed, do you want to expunge it?")
```
Could you test this for me?